Manager/src/main/java/io/raspberrywallet/manager/cryptography/crypto/algorithms/AESParameters.java

```diff
@@ -1,30 +1,30 @@
-package io.raspberrywallet.manager.cryptography.ciphers;
+package io.raspberrywallet.manager.cryptography.crypto.algorithms;
 
-public class AESFactory extends AlgorithmFactory {
+class AESParameters extends AlgorithmFactory {
```

Comment:
`AESParametets extends AlgorithmFactory` - doesn't make sense. `AlgorithmFactory` should me renamed to `AlgorithmParameters`